docs/ingestion/ingestion-spec.md

```diff
@@ -200,6 +212,23 @@ configuring [dimensions](./data-model.md#dimensions). An example `dimensionsSpec
 }
 ```
 
+<!--Schema auto-discovery-->
+
+```json
+    "tuningConfig": {
+      ...
+      "appendableIndexSpec": {
+        ...,
+        "useSchemaDiscovery": true
+      },
+    },
```

Comment:
oops, the old PR descriptions are stale, `useSchemaDiscovery` is on `dimensionsSpec` these days

Reply:
Is it still a child of `appendableIndexSpec`, ie:
`dimensionsSpec.appendableIndexSpec.useSchemaDiscovery`?

Reply:
no, just directly on `dimensionsSpec`, `appendableIndexSpec` is part of `tuningConfig`